.dockerignore

```diff
@@ -0,0 +1,8 @@
+.git
```

Comment:
Putting this here because it's the top, please add some information in the README about this image and how to deploy it and the versioning I described with frequency below